### PR TITLE
chore(terraform): switch to pkgs.k8s.io

### PR DESCRIPTION
**Pull request by tiagolobocastro**
https://kubernetes.io/blog/2023/08/15/pkgs-k8s-io-introduction/

### CODE REVIEW

File: terraform/cluster/variables.tf [hunk 1]
```diff
@@ -103,7 +103,7 @@ variable "master_vcpu" {
 variable "kubernetes_version" {
   type        = string
   description = "Version of all kubernetes components"
-  default     = "1.26.0-00"
+  default     = "1.26.14-1.1"
```
Comment (niladrih) [inline terraform/cluster/variables.tf:106]: Is this a valid version?

**Review by niladrih — APPROVED** [review]
It's crazy that kubernetes has a dpkg package repository!

File: terraform/cluster/variables.tf (hunk 1 above)
Comment (tiagolobocastro) [reply terraform/cluster/variables.tf:106]: It is indeed

**Comment by tiagolobocastro** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors cancel

**Comment by tiagolobocastro** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors cancel

**Comment by tiagolobocastro** [comment]
bors merge

**Pull request merged**